### DISCUSSION TITLE
Any way to add loading status to a div that is not form or segment?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1882] [Elements (button, image, label, loader, ...)] [answered]
Hello,
often I use "loading" class in forms (`<div class="ui loading form">`) and segments (`<div class="ui loading segment">`) in my Fomantic-UI applications.

And often I need to have loading in other `div`, that are not form or segment, as well.

Any way to do that?

I have tried to add a segment "fitted" parent cointainer to my existing div or to use "ui segment fitted" classe directly on the given div, but it adds borders and I have to remove it throught CSS.

Any better solution?

**Comment by lubber-de** [comment, marked as answer]
You can use your approach but a `ui basic dimmer` which does not render a border
 You my also think of using a separate `ui loader` instead. 
https://fomantic-ui.com/elements/loader.html

`loading` is a dedicated variant to some components.
